llama-index-packs/llama-index-packs-secgpt/pyproject.toml

```diff
@@ -0,0 +1,77 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+# Feel free to un-skip examples, and experimental, you will just need to
+# work through many typos (--write-changes and --interactive will help)
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.llamahub]
+contains_example = true
+import_path = "llama_index.packs.secgpt"
+
+[tool.llamahub.class_authors]
+Hub = "Yuhao-W"
+HubOperator = "Yuhao-W"
+HubPlanner = "Yuhao-W"
+Message = "Yuhao-W"
+Socket = "Yuhao-W"
+Spoke = "Yuhao-W"
+SpokeOperator = "Yuhao-W"
+SpokeOutputParser = "Yuhao-W"
+TIMEOUT = "Yuhao-W"
+ToolImporter = "Yuhao-W"
+VanillaSpoke = "Yuhao-W"
+create_function_placeholder = "Yuhao-W"
+create_message_spoke_tool = "Yuhao-W"
+drop_perms = "Yuhao-W"
+get_user_consent = "Yuhao-W"
+set_mem_limit = "Yuhao-W"
```

Comment:
all of these will show up in `llamahub.ai` which i don't think is ideal. Probably only makes sense to have the main one be discoverable i.e., `Hub`.

Reply:
It makes sense to me. I have revised.